cmd/telemetry.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"runtime"
+)
+
+type metric struct {
+	Source string `json:"source,omitempty"`
+	Action string `json:"action,omitempty"`
+	Error  string `json:"error,omitempty"`
+	OS     string `json:"os,omitempty"`
+	Arch   string `json:"arch,omitempty"`
+}
+
+const (
+	telemetryURI = "https://telemetry.harbor.turnerlabs.io/v1/api/metric"
```

Comment:
drive from an environment variable for ease of change in the future?

Reply:
ya, good point